.github/workflows/auto_docs.yml

```diff
@@ -0,0 +1,30 @@
+name: Automatic Documentation for Launcher
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update_output:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    
+    - name: Run TGI Launcher docs
+      id: run-launcher-help
+      run: |
+        output=$(text-generation-launcher --help)
```

Comment:
I don't think the GitHub Actions workflow has anything installed to run `text-generation-launcher --help`. You will need to either install this in GH Actions or use a Docker GA that allows you to do `docker run ghcr.io/huggingface/text-generation-inference:latest --help` (which I think has same output)

Reply:
Maybe include it in the `build` step too (because there it's available as the docker image)

Reply:
ah right. @Narsil is `docker run` more lightweight compared to only building launcher? 
```
cd ../../launcher
cargo build
```

Reply:
I added the above two lines to action file.

Reply:
I realized `make install` works better for this case.